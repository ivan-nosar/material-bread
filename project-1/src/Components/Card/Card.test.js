import React from 'react';
import { Card } from '../..';

import renderer from 'react-test-renderer';

test('Card Renders', () => {
  const tree = renderer.create(<Card />).toJSON();
  expect(tree).toMatchSnapshot();
});
