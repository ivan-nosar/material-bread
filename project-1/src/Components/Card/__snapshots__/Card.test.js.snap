// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Renders 1`] = `
<View
  style={
    Object {
      "backgroundColor": "white",
      "borderBottomColor": "rgba(0,0,0,.4)",
      "borderRadius": 4,
      "borderWidth": 0,
      "shadowColor": "#000",
      "shadowOffset": Object {
        "height": 1,
        "width": 0,
      },
      "shadowOpacity": 0.18,
      "shadowRadius": 1,
    }
  }
/>
`;
